src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {

```

Comment:
That's not what you should put there - you should be more precise. If it's BulldozerProducer, then what should be type parameter? Fix in all producers